<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AxisRangeDialog</class>
 <widget class="QDialog" name="AxisRangeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>156</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>156</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>156</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Set coordinates ranges</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="x_label">
       <property name="text">
        <string>X range from: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="xFromLineEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="x_defis_label">
       <property name="text">
        <string>- to: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="xToLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="y_label">
       <property name="text">
        <string>Y range from: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="yFromLineEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="y_defis_label">
       <property name="text">
        <string>- to: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="yToLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okAxRangeButton">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
